libvirtd-desktop/files/etc/systemd/system/virtlogd.service.d/override.conf

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2025 Cody Istre
+# SPDX-License-Identifier: MIT
```

Comment:
Let's move that in https://github.com/fedora-sysexts/fedora/blob/main/REUSE.toml#L20 as it's not even sure if this is copyright worthy.